packages/adapter/lib/adapter/adapter.js

```diff
@@ -8531,7 +8533,7 @@ function Adapter(options) {
                                 logger.error(this.namespaceLog + ' Cannot load node-schedule. Scheduled restart is disabled');
                             }
                             if (schedule) {
-                                logger.debug(this.namespaceLog + ' Schedule restart: ' + adapterConfig.common.restartSchedule);
+                                logger.silly(`${this.namespaceLog} Schedule restart: ${adapterConfig.common.restartSchedule}`);
```

Comment:
I would change that back to debug because it is an important info in "why my adapter restarts that often" cases because people configured a "restart every minute"
```suggestion
                                logger.debug(`${this.namespaceLog} Schedule restart: ${adapterConfig.common.restartSchedule}`);
```